cachito/web/models.py

```diff
@@ -172,18 +172,21 @@ def to_json(self):
 
         env_vars_json = OrderedDict(env_var.to_json() for env_var in self.environment_variables)
         rv = {
-            'dependencies': [dep.to_json() for dep in self.dependencies],
             'id': self.id,
             'repo': self.repo,
             'ref': self.ref,
             'pkg_managers': pkg_managers,
-            'state_history': states,
             'user': user,
             'environment_variables': env_vars_json,
             'flags': [flag.to_json() for flag in self.flags if flag.active],
         }
         # Show the latest state information in the first level of the JSON
         rv.update(latest_state)
+        if verbose:
```

Comment:
Are there some users of existing API already? If so, are we sure the (by default) removed/changed `dependencies` and `state_history` won't break them?

Reply:
The API is not used yet which is why we're making this change now  :smile: